releases/2018.XX.md

```diff
@@ -7,6 +7,7 @@
 * Added $preview variable which is set to true in preview mode (#1963)
 * Removed glide() and subdiv() which were never implemented
 * Added is_undef() to look up variables silently (#2535)
+* Added is_list(), is_num, is_bool, is_string
```

Comment:
Probably add `()` behind all functions for consistency

Reply:
thank you for the input - updated